.github/workflows/pages.yml

```diff
@@ -41,7 +44,7 @@ jobs:
       - name: Setup Pages
         id: pages
         uses: actions/configure-pages@v1
-      - run: bundle exec jekyll build
+      - run: bundle exec jekyll build --baseurl "${{env.BASE_URL}}"
```

Comment:
I needed to do this to make the site display correctly on my fork. This _shouldn't_ mess up anything in the main repo; the BASE_URL will just be an empty string.

Reply:
I feel like I want to do this at a time where we can do a rollback given what we saw on Friday; is that ok?

Reply:
Yep, but I'd like to merge the other stuff in this PR sooner so I'll pull this change out of the PR